src/lib/loaders/api/loader_interface.ts

```diff
@@ -60,29 +60,29 @@ const encodeDynamicPath = (
  */
 
 export function loaderInterface<T>(
+  method: string,
   loader: DataLoader<DataLoaderKey, T>,
   pathOrGenerator: string,
-  globalParams: any,
-  method?: string
+  globalParams: any
 ): StaticPathLoader<T>
 
 export function loaderInterface<T, P>(
+  method: string,
   loader: DataLoader<DataLoaderKey, T>,
   pathOrGenerator: PathGenerator<P>,
-  globalParams: any,
-  method?: string
+  globalParams: any
 ): DynamicPathLoader<T>
 
 export function loaderInterface<T, P>(
+  method = "GET",
   loader: DataLoader<DataLoaderKey, T>,
   pathOrGenerator: string | PathGenerator<P>,
-  globalParams: any,
-  method?: string
+  globalParams: any
 ) {
   const dynamicPath = typeof pathOrGenerator === "function"
   const keyGenerator: any = dynamicPath ? encodeDynamicPath : encodeStaticPath
   return (...args) => {
-    const key = keyGenerator(pathOrGenerator, globalParams, ...args, method)
```

Comment:
This was the bug, can't have `method` come at the end after the `...args` spread catch-all.